diff -urp astropy-healpix-0.4.orig/astropy_healpix/conftest.py astropy-healpix-0.4/astropy_healpix/conftest.py
--- astropy-healpix-0.4.orig/astropy_healpix/conftest.py	2018-12-19 04:23:52.000000000 +0900
+++ astropy-healpix-0.4/astropy_healpix/conftest.py	2019-09-13 22:49:01.818401095 +0900
@@ -4,7 +4,11 @@
 # by importing them here in conftest.py they are discoverable by py.test
 # no matter how it is invoked within the source tree.
 
-from astropy.tests.pytest_plugins import *
+try:
+    from astropy.tests.plugins.display import *
+    from astropy.tests.helper import *
+except ImportError:
+    from astropy.tests.pytest_plugins import *
 
 # Uncomment the following line to treat all DeprecationWarnings as
 # exceptions. For Astropy v2.0 or later, there are 2 additional keywords,
